circle-animation.js

```diff
@@ -0,0 +1,49 @@
+class Circle {
+    constructor(x, y, size) {
+      this.x = x;
+      this.y = y;
+      this.size = size;
+      this.color = color(random(255), random(255), random(255), 200);
```

Comment:
you need to define these attributes on the class itself, not only in the constructor